src/cow-react/modules/application/containers/App/Updaters.tsx

```diff
@@ -46,7 +45,6 @@ export function Updaters() {
       <SentryUpdater />
       <UploadToIpfsUpdater />
       <AppDataUpdater />
-      <GnosisSafeUpdater />
```

Comment:
we don't need a specific updater for the safe. The `WalletUpdater` will include it in its responsability